.github/workflows/publish-cli.yml

```diff
@@ -123,7 +123,7 @@ jobs:
         run: |
           gon gon-config.json
           unzip wallet.zip
-          mv -f wallet target/production/wallet
+          mv -f wallet ../target/production/wallet
```

Comment:
Oh then we can also remove the `working-directory: cli`, no? I added it because I thought the cli had an own workspace

Reply:
I think in this case we may need `working-directory: cli`. If we run `gon` with the root of the repo as the working directory and unzip `wallet.zip`, it'll create a file in the repo root called `wallet` and that will conflict with the `wallet` workspace folder